src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/CoveragePoNNormalizationResult.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.hellbender.tools.pon.coverage;
+
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+
+/**
+ * Interface for a coverage profile that has been normalized by a {@link CoveragePanelOfNormals}.
+ */
+public interface CoveragePoNNormalizationResult {
+    ReadCountCollection getProfile();
```

Comment:
will add it for now, but I wonder if we will want to add any methods to this interface in the future?